status/views.py

```diff
@@ -57,7 +60,7 @@ def status(request, username=None):
             user = User.objects.get(username=username)
             submissions = submissions.filter(user=user)
         except:
-            raise Http404('User %s Not Found!' % username)
+            raise Http404
```

Comment:
I prefer to redirect to 500 error page, with error message "user doesn't exist".